src/files/delete-prompt/DeletePrompt.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import TrashIcon from '../../icons/StrokeTrash'
+import CancelIcon from '../../icons/GlyphSmallCancel'
+import Button from '../../components/button/Button'
+
+const DeletePrompt = ({cancel, action, folders, files, className, ...props}) => {
+  className = `${className} w-80 shadow-4 sans-serif relative`
+  let what
+
+  if (folders > 0) {
+    if (files > 0) {
+      what = 'Items'
+    } else {
+      what = (folders === 1) ? 'Folder' : 'Folders'
+    }
+  } else {
+    what = (files === 1) ? 'File' : 'Files'
+  }
+
+  return (
+    <div className={className} style={{maxWidth: '30em'}} {...props}>
+      <CancelIcon className='absolute pointer w2 h2 top-0 right-0 fill-gray' onClick={cancel} />
+
+      <div className='ph2 pv3 tc'>
+        <div className='center bg-snow br-100 flex justify-center items-center' style={{width: '80px', height: '80px'}}>
+          <TrashIcon className='fill-gray w3' />
+        </div>
+
+        <p className='charcoal-muted fw5'>Delete {what}</p>
+
+        <p className='gray w-80 center'>
+          Are you sure you want to delete this {what.toLowerCase()}? This action is permanent and cannot be reversed.
```

Comment:
singular: 'this folder' 
plural: 'these folders'

same for items, and files... perhaps it's time to get https://github.com/yahoo/react-intl involved?